pkg/framework/entutils/entexpose/entexpose.go

```diff
@@ -0,0 +1,26 @@
+package entexpose
```

Comment:
nit: i'd keep `entexpose` and `entpaginate` in the same package, maybe simply `extensions`

Reply:
I prefer them separate, as nit I'll skip on it